device/arista/x86_64-arista_7260cx3_64/Arista-7260CX3-C64/config.bcm.j2

```diff
@@ -11,17 +12,19 @@
 {%-     if DEVICE_METADATA['localhost']['subtype'] is defined -%}
 {%-         set switch_subtype = DEVICE_METADATA['localhost']['subtype'] -%}
 {%-         if 'dualtor' in switch_subtype.lower() %}
-{%-             set IPinIP_sock = 'sai_tunnel_support=1
-                                   sai_tunnel_underlay_route_mode=1
-                                   host_as_route_disable=1
-                                   l3_ecmp_levels=2' -%}
+{%-             set IPinIP_sock = 'sai_tunnel_support=1\r\n'
+                                   + 'sai_tunnel_underlay_route_mode=1\r\n'
```

Comment:
why do we have extra + here?

Reply:
I suggest to keep original pr, do not like this \r\n, looks like a unix dos issue.

Reply:
revert the change